aip/general/0180.md

```diff
@@ -159,7 +159,7 @@ this guidance could ostensibly prevent _any_ change (which is not the intent).
 [aip-151]: ./0151.md
 [aip-158]: ./0158.md
 [aip-181]: ./0181.md
-[aip-4231]: ./client-libraries/4231.md
-[aip-4232]: ./client-libraries/4232.md
+[aip-4231]: ./4231
```

Comment:
It's odd that the other links are ".md" and these aren't. Does the publication pipeline "know" what to do with all of these? Do we have any documentation for how links are meant to appear? (Could we automatically know that `[AIP-4231]` should go to AIP 4231 without even adding these links?)

Reply:
Well, I have ran the service locally and it seems the only way this works is when ".md" is not present when there are files referenced from subdirectories. I suspect we can update the generator but for the time being this seems to work.